packages/studio-plugin/package.json

```diff
@@ -33,7 +33,7 @@
     "@babel/preset-env": "^7.22.4",
     "@babel/preset-react": "^7.22.3",
     "@babel/preset-typescript": "^7.21.5",
-    "@yext/pages": "^1.0.0-beta.14",
+    "@yext/pages": "^1.0.0-rc.1",
```

Comment:
I think the third party notices for `studio-plugin` need to be updated to reflect this change